recipes/trinotate/meta.yaml

```diff
@@ -0,0 +1,45 @@
+about:
+  home: https://trinotate.github.io/
+  license: ''
+  summary: 'Trinotate is a comprehensive annotation suite designed for automatic
+    functional annotation of transcriptomes, particularly de novo assembled
+    transcriptomes, from model or non-model organisms'
+
+build:
+  number: 1
+  skip: True # [osx]
+
+package:
+  name: trinotate
+  version: '3.2.2'
+
+requirements:
+  build:
+      - libgcc
```

Comment:
libgcc does not need to be an build requirement. It can be a run-requirement. Also please double check if you really need the gcc as dependency at all. Isn't this just a perl script?
Can we try to remove the skip-osx step?

Thanks a lot for working on this. It's also on my todo list for quite some time :)